Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -71,6 +87,14 @@ namespace species
             return 0.;
         case PhysicalSpecies::hydrogen:
             return PhysConst::m_p;
+        case PhysicalSpecies::carbon:
+            return PhysConst::m_e * amrex::Real(22032.0);
+        case PhysicalSpecies::nitrogen:
+            return PhysConst::m_e * amrex::Real(25716.9);
+        case PhysicalSpecies::oxygen:
+            return PhysConst::m_p * amrex::Real(15.8834);
+        case PhysicalSpecies::copper:
+            return PhysConst::m_p * amrex::Real(63.0864);
```

Comment:
Is there a particular reason for expressing some masses as a function of  `m_p` and some others as a function of `m_e` ?

Reply:
No, I just copied from a prior input I used in that other PIC on GPU code of mine :sweat_smile: